components/card/package.json

```
@@ -23,18 +23,18 @@
     "@spectrum-css/checkbox": "^3.0.0 || ^4.0.0 || ^5.0.0 || ^6.0.0",
     "@spectrum-css/icon": "^3.0.0",
     "@spectrum-css/quickaction": "^3.0.0",
-    "@spectrum-css/typography": "^4.0.0",
-    "@spectrum-css/vars": ">=9.0.0"
+    "@spectrum-css/tokens": "^8.0.1",
+    "@spectrum-css/typography": "^4.0.0"
```

```suggestion
    "@spectrum-css/tokens": ">=8 <=9",
    "@spectrum-css/typography": ">=3 <=4"
```

Should we move to making peer dependencies more permissive to allow more cross-component compatibility?

---

I think that makes sense. We have a bunch of or's going on for the other peer dependencies so this could also clean that up a bit